Existem várias Exceptions

try:
	operação
except:
	fazer algo se der errado
else:
	Fazer algo se deu certo
finally:
	print(“Volte Sempre”) # sempre ocorre

EX:
try:
	a = int(input(‘Numerador: ’))
	b = int(input(‘Denominador: ’))
	r = a/b
except (ValueError, TypeError):
	print(‘Tivemos um problema com os tipos de dados que você digitou.’)
except ZeroDivisionError:
	print(‘Não possível dividir um número por zero.’)
except Exception as erro:
	print(f‘O erro encontrado foi {erro.__cause__}’)
else:
	print(f‘O resultado é {r:.1f}’)
finally:
	print(‘Volte Sempre!’)
